test/sign_blob_test.sh

```diff
@@ -29,11 +29,11 @@ BLOB=verify-experimental-blob
 date > $BLOB
 cat $BLOB
 
-echo "Sign the blob with cosign first and upload to rekor"
-SIG=$($COSIGN_CLI sign-blob $BLOB)
+# echo "Sign the blob with cosign first and upload to rekor"
```

Comment:
think these were commented out as part of debugging?

Reply:
opssy :)

Reply:
done